fedmsg_meta_umb/tests/test_distgit.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# fedmsg_meta_umb is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# fedmsg_meta_umb is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with fedmsg; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Authors:  Mike Bonnet <mikeb@redhat.com>
+
+import fedmsg.tests.test_meta
+
+
+class TestDistGit(fedmsg.tests.test_meta.Base):
+    """ The Dist-Git system manages collections of git repositories.
+
+    Messages are published every time one or more commits are pushed to a repo.
+    """
+    expected_title = 'distgit.commit'
+    expected_subti = '42bcbca2 was committed on the rhel-7.4 branch of the rpcbind rpm repo by steved'
+    expected_link = ('https://pkgs.devel.redhat.com/cgit/rpms/rpcbind/commit/'
+                     '?h=rhel-7.4&id=42bcbca202b35ae663bdbca9e10b64d4a5dfbf5a')
+    expected_packages = set(['rpcbind'])
```

Comment:
Should add ``expected_usernames`` and ``expected_agent`` here since you know about a *user* associated with this event.